omgeo/postprocessors.py

```diff
@@ -329,6 +329,50 @@ def __repr__(self):
             (self.__class__.__name__, self.is_exact(), self.attr, self.bad_values)
 
 
+class AttrListIncludes(_PostProcessor):
+    """
+    PostProcessor used to ditch results without desired attribute list items.
```

Comment:
This comment should differ from that for `AttrFilter` What does it do that is different?

Reply:
The key difference is in the last two words - it operates on attributes that are lists instead of scalars (and similar for `AttrListExcludes`). Happy to rework it further if you have any specific suggestions.